### ISSUE TITLE
Q: Do we use the same sentence from Twitter to Reddit?

### ISSUE THREAD

**Issue by cyruslk** [cyruslk/franklin-ford-bot#8]
I guess we do use the same sentence but we compute it differently but just want to make sure with you. I guess using the same sentence is more interesting because then we have both a platform where this sentence is outputted without any targeting (Twitter, no #), then we have the propagation of this same sentence on Reddit; something we could easily use for for visualisation (I'll start thinking about this soon!) inside the bot's website.

**Comment by juliettedm** [comment]
Yes we can use the same sentences. I'm also interested in seeing how the same sentences "work" differently on different platforms.
